printCardImages.py

```diff
@@ -30,24 +30,23 @@ async def print_card_images(message: Message):
                 allCards[post].getImg(), allCards[post].getName(), message
             )
 
-
-async def sendImageReply(url: str, cardname: str, message: Message):
+async def sendImageReply(url: str, cardname: str, text: str, message: Message):
     print(f"sendimage{message.author.id}")
     async with aiohttp.ClientSession() as session:
         async with session.get(url) as resp:
             if resp.status != 200:
                 await message.reply(
                     "Something went wrong while getting the link for "
                     + cardname
-                    + ". Wait for llllll to fix it."
+                    + ". Wait for llllll or klunker to fix it."
                 )
                 await session.close()
                 return
-            # currently extraFilename looks like inline;filename="                                Skald.png"
+            # currently extraFilename looks like inline;filename="Skald.png"
             extraFilename = resp.headers.get("Content-Disposition")
             parsedFilename = re.findall('inline;filename="(.*)"', str(extraFilename))[0]
             data = io.BytesIO(await resp.read())
-            sentMessage = await message.reply(
+            sentMessage = await message.reply(text,
```

Comment:
one thing i've picked up is never trusting indexed args, mind doing a content=text here?